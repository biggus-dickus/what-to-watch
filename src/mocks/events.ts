// JSDom has no implementation for common methods of HTML media elements.
// Tests will fail without this. Sad but true.
// https://github.com/jsdom/jsdom/issues/2155#issuecomment-366703395
export const mockHTMLMediaMethods = (): void => {
  window[`HTMLMediaElement`].prototype.load = () => { /* do nothing */ };
  window[`HTMLMediaElement`].prototype.play = () => { /* do nothing */ };
  window[`HTMLMediaElement`].prototype.pause = () => { /* do nothing */ };
};
