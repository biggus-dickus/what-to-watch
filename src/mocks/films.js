export const films = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    picUrl: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Bohemian Rhapsody`,
    picUrl: `img/bohemian-rhapsody.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Macbeth`,
    picUrl: `img/macbeth.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Aviator`,
    picUrl: `img/aviator.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `We need to talk about Kevin`,
    picUrl: `img/we-need-to-talk-about-kevin.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `What We Do in the Shadows`,
    picUrl: `img/what-we-do-in-the-shadows.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Revenant`,
    picUrl: `img/revenant.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Johny English`,
    picUrl: `img/johnny-english.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Shutter Island`,
    picUrl: `img/shutter-island.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Pulp Fiction`,
    picUrl: `img/pulp-fiction.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `No Country for Old Men`,
    picUrl: `img/no-country-for-old-men.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Snatch`,
    picUrl: `img/snatch.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Moonrise Kingdom`,
    picUrl: `img/moonrise-kingdom.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Seven Years in Tibet`,
    picUrl: `img/seven-years-in-tibet.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Midnight Special`,
    picUrl: `img/midnight-special.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `War of the Worlds`,
    picUrl: `img/war-of-the-worlds.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Dardjeeling Limited`,
    picUrl: `img/dardjeeling-limited.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Orlando`,
    picUrl: `img/orlando.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Mindhunter`,
    picUrl: `img/mindhunter.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
];
