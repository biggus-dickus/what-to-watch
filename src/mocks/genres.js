export const Genre = {
  ALL: `All genres`,
  COMEDIES: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMAS: `Dramas`,
  HORROR: `Horror`,
  FAMILY: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLERS: `Thrillers`
};
