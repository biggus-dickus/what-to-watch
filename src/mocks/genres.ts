import {Genre as GenreType} from '../types'; // eslint-disable-line

export const Genre = {
  ALL: `All genres`,
  COMEDIES: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMAS: `Dramas`,
  HORROR: `Horror`,
  FAMILY: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLERS: `Thrillers`
};

export const mockGenres = Object.values(Genre);

export const mockGenreEntity: GenreType = {
  currentGenre: Genre.CRIME,
  genres: [...mockGenres],
  onGenreChange: jest.fn()
};
