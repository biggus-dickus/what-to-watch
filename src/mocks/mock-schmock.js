export const mockGenres = [`All`, `BDSM`, `Gay gymnastics`, `Psycho-dramas for aestheticizing faggots`];

export const mockFilms = [
  {
    name: `L'elephant vert`,
    genre: [mockGenres[1], mockGenres[2]],
    pageUrl: `https://www.imdb.com/title/tt1003080/`,
    picUrl: `http://lurkmore.so/images/4/47/Slonik_macro.jpg`,
    videoUrl: `https://storage.googleapis.com/coverr-main/mp4%2FCam-Man.mp4`
  },
  {
    name: `Sal√≤ o le 120 giornate di Sodoma`,
    genre: [mockGenres[3]],
    pageUrl: `https://www.imdb.com/title/tt0073650/`,
    picUrl: `https://m.media-amazon.com/images/M/MV5BYjE0NTU3MTUtYjE0OC00ZGM1LWIwNzYtNDU2YWI2YWJjZjEwXkEyXkFqcGdeQXVyMTA2ODMzMDU@._V1_SY1000_CR0,0,764,1000_AL_.jpg`,
    videoUrl: `etc/na-zdorovnya.mp4`
  },
  {
    name: `Un Chien Andalou`,
    genre: [mockGenres[3]],
    pageUrl: `https://www.imdb.com/title/tt0020530/`,
    picUrl: `https://m.media-amazon.com/images/M/MV5BMjZlY2RjM2ItZGEwOC00ODQwLWFhZDYtMjE3ODMyYWNiOWNlXkEyXkFqcGdeQXVyNTA4NzY1MzY@._V1_SY1000_CR0,0,706,1000_AL_.jpg`,
    videoUrl: `etc/na-zdorovnya.mp4`
  }
];
