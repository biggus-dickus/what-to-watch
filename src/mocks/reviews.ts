import {Review} from '../types'; // eslint-disable-line

export const mockReviews: Review[] = [
  {
    id: 1,
    user: {
      id: 4,
      name: `Kate Muir`,
    },
    rating: 8.9,
    comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
    date: `2019-05-08T14:13:56.569Z`
  },
  {
    id: 2,
    user: {
      id: 13,
      name: `Bill Goodykoontz`,
    },
    rating: 8.0,
    comment: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight. “The Grand Budapest Hotel” is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
    date: `2019-06-07T21:12:21.569Z`
  }
];
