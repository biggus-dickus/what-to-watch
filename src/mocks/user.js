const mockUser = {
  id: 1,
  email: `Oliver.conner@gmail.com`,
  name: `Oliver.conner`,
  avatar_url: `img/1.png` // eslint-disable-line
};

export default mockUser;
