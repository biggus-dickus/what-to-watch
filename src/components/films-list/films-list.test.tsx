import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {StaticRouter as Router} from 'react-router-dom';
import * as Adapter from 'enzyme-adapter-react-16';
import {configure, shallow} from 'enzyme';

import {mockFilms} from '../../mocks/films';

import FilmsList from './films-list';

configure({adapter: new Adapter()});


describe(`FilmsList test suite`, () => {
  it(`<FilmsList /> should render correctly`, () => {
    const tree = renderer.create(<Router><FilmsList films={mockFilms} /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should slice received films on mount according to the limit set in config or props and display the "Show more" button`, () => {
    const wrapper = shallow(<FilmsList films={mockFilms} limitTo={4} />);

    expect(wrapper.find(`[data-test="at-films-list"]`).children()).toHaveLength(4);
    expect(wrapper.find(`[data-test="at-films-show-more"]`)).toHaveLength(1);
  });

  it(`should NOT display the "Show more" button if the length of supplied films is less than the slice limit`, () => {
    const wrapper = shallow(<FilmsList films={mockFilms} />);
    expect(wrapper.find(`[data-test="at-films-show-more"]`)).toHaveLength(0);
  });
});
