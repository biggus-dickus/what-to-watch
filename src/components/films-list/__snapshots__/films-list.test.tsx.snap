// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsList test suite <FilmsList /> should render correctly 1`] = `
<div
  className="catalog__movies-list"
  data-test="at-films-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
          src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Fantastic Beasts: The Crimes of Grindelwald
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/bohemian-rhapsody.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Bohemian Rhapsody
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/macbeth.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4%2FCam-Man.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Macbeth
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/aviator.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/5/5b/Marnie_%281964%29_trailer.webm/Marnie_%281964%29_trailer.webm.360p.vp9.webm"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Aviator
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/we-need-to-talk-about-kevin.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/5/5b/Marnie_%281964%29_trailer.webm/Marnie_%281964%29_trailer.webm.360p.vp9.webm"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          We need to talk about Kevin
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/what-we-do-in-the-shadows.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/2/23/Dr_Strangelove_-_Official_Trailer.webm/Dr_Strangelove_-_Official_Trailer.webm.480p.vp9.webm"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          What We Do in the Shadows
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/revenant.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Harvesting%20Rice.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Revenant
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/johnny-english.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Lake%20Day.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Johny English
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/shutter-island.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Deserted%20Island.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Shutter Island
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/pulp-fiction.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Stunning%20Mountains.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Pulp Fiction
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/no-country-for-old-men.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Gubat%20Beach.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          No Country for Old Men
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/snatch.jpg"
          src="etc/na-zdorovnya.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Snatch
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/moonrise-kingdom.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Caramoan%20Drone.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Moonrise Kingdom
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/seven-years-in-tibet.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Ants%20Walking.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Seven Years in Tibet
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/midnight-special.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Boat%20in%20Caramoan.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Midnight Special
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/war-of-the-worlds.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Blue%20Joy.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          War of the Worlds
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/dardjeeling-limited.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Stop-Sign.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Dardjeeling Limited
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/orlando.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/Steaming-Yellowstone.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Orlando
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    tabIndex={0}
  >
    <a
      className="small-movie-card__wrapping-link"
      href="/https://thepiratebay.org"
      onClick={[Function]}
    >
      <i
        aria-hidden={true}
        className="small-movie-card__play-btn"
      />
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          muted={true}
          poster="img/mindhunter.jpg"
          src="https://storage.googleapis.com/coverr-main/mp4/California-Desert.mp4"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Mindhunter
        </span>
      </h3>
    </a>
  </article>
</div>
`;
